src/main/resources/application.yml

```diff
@@ -1,3 +1,9 @@
+spring:
+  flyway:
+    enabled: true
+    baselineOnMigrate: true
+    clean-on-validation-error: true
```

Comment:
`Warning ! Do not enable in production !`

Reply:
Nice catch fikser det!